pkg/pagerduty/pagerduty.go

```diff
@@ -0,0 +1,100 @@
+package pagerduty
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+
+	pkgerrors "github.com/pkg/errors"
+
+	sdk "github.com/PagerDuty/go-pagerduty"
+)
+
+const (
+	InvalidInputParamsErrorCode = 2001
```

Comment:
I am not a big fan of redefining magic constants that are used by other code. Can we not import this? Or do we really need this?

Reply:
this is the error code that was pulled from the PD api, I can explain where I got it if that helps

Reply:
yes at least a link or so added as a comment sounds good to me

Reply:
I cannot provide a link as I found the error during running the command manually. and the API page doesn't note this at all

Reply:
[https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTYz-errors](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTYz-errors) search for code '2001'